tool/python/singa/layer.py

```diff
@@ -228,21 +228,20 @@ def __init__(self, size=0, **kwargs):
 
 class Activation(Layer):
 
-    def __init__(self, activation='stanh', topk=1):
+    def __init__(self, activation='stanh'):
         '''
         required
-          activation = (string)
-        optional
-          topk       = (int)  // the number of results
+          activation = (string) // relu, sigmoid, tanh, stanh, softmax.
         '''
 
+        if activation == 'tanh':
+          print 'Warning: Tanh layer is not supported for CPU'
+
         self.name = activation
-        if activation == 'tanh': activation = 'stanh' # <-- better way to set?
         self.layer_type = enumLayerType(activation)
```

Comment:
Can I add "kActivation" enum in LayerType for tanh/relu/sigmoid ?

Then, I will set the ActivationLayer type to TANH/ReLU/SIGMOID
In this way, it will be clearer to distinguish cpu and gpu

Later, stanh will follow the same manner after updating ActivationLayer to enable STANH